aplib.net-demo/Assets/Scripts/MainMenu/VolumeSlider.cs

```diff
@@ -0,0 +1,48 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// This class manages the bar and master volume.
+/// </summary>
+public class VolumeSlider : MonoBehaviour
+{
+    /// <summary>
+    /// Used to set the volume bar to the correct repesentation.
+    /// </summary>
+    Slider _volSlider;
+
+    /// <summary>
+    /// Used to call and store volume value.
+    /// </summary>
+    float _volumeValue;
+
+    /// <summary>
+    /// Takes the value from the volume slider and sends it to the sound manager.
+    /// </summary>
+    /// <param name="newValue"></param>
+    public void VolumeValue(float newValue)
+    {
+        _volumeValue = newValue;
+        // To Do:: Send newValue that is from 1 to 100 to the Sound manager
```

Comment:
Why not do this now? If not, have you notified Joachim that a ticket for this must be created
```suggestion
        // TODO:: Send newValue that is from 1 to 100 to the Sound manager
```

Reply:
Yes, I have notified Joachim and if I remember right this will be fixed with the sounds/animations. Right now, all sounds are not centralized and it will be a pain fixing this.